azdo-task/scripts/build-package.sh

```diff
@@ -54,9 +54,9 @@ echo "VERSION_MAJOR=${VERSION_MAJOR}"
 echo "VERSION_MINOR=${VERSION_MINOR}"
```

Comment:
@stuartleeks these are the lines where the version is not grabbed which causes build job to fail, they should not be failing due to anything I changed because this is unrelated code. On occasion it runs, and on occasion it does not.

Reply:
My mistake, I think it may actually be the lines I added for changing output method:

```sh
  [2023-03-03T12:20:46.703Z] --set-patch-version not set
  
  [2023-03-03T12:20:46.717Z] VERSION_MAJOR=0
  VERSION_MINOR=3
  
  [2023-03-03T12:20:46.717Z] 
  VERSION_PATCH=0
  version=0.3.0 >> 
  version_short=0.3 >> 
  version_major=0 >> 
  ```
  
  I thought the required version was 1.1.0 or greater for actions/core, but maybe I missread and .10 is needed for this as you have set in #221

Reply:
I did misread, I have updated quotes to be appropriate for command, and will wait for #221 to be merged to test again.